helm/nifikop/values.yaml

```diff
@@ -33,6 +33,9 @@ resources:
 ## Ref: https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/
 hostAliases: []
 
+## RunAsUser for OpenShift compatibility
+runAsUser: 1001
```

Comment:
The default should probably stay as `1000` to keep functionality the same?

Reply:
totally agree my bad 1001 is an OpenShift convention uid.